app/models/apple/show.rb

```diff
@@ -2,15 +2,17 @@
 
 module Apple
   class Show
-    attr_reader :feed, :api
+    attr_reader :public_feed,
```

Comment:
Could also change this to have `:apple_credential` instead, and pull the public/private feed from that?

But this also works.  And is maybe cleaner to test.